DependencyInjection/CompilerPass/RegisterCompilerPass.php

```diff
@@ -53,11 +59,12 @@ public function process(ContainerBuilder $container)
      *
      * @param string $actionDirectory
      *
-     * @return string[]
+     * @return array
      */
     private function getClasses($actionDirectory)
```

Comment:
I just realized that this argument name should be changed to something more generic like ``$directory``.